bokehjs/src/lib/models/widgets/tables/data_table.ts

```diff
@@ -183,16 +183,10 @@ export class DataTableView extends WidgetView {
 
   override connect_signals(): void {
     super.connect_signals()
-    this.connect(this.model.change, () => {
-      this.render()
-      this.r_after_render()
-    })
+    this.connect(this.model.change, () => this.rerender())
 
     for (const column of this.model.columns) {
-      this.connect(column.change, () => {
-        this.render()
-        this.r_after_render()
-      })
+      this.connect(column.change, () => this.rerender())
```

Comment:
Is `renderer` a good name here? Its a noun, not a verb, so I'd expect it e.g. to return some object or thing, not "do" something (which is what I would expect an event callback to do)

Reply:
I'm pretty sure "re-render" is a verb, but I suppose it depends how you use it in a sentence. To me it's clear this is an action to take, the same as "redo", etc.

Reply:
Sorry I literally just misread it since "renderER" is already so common in the codebase. `re_render` would help but is a bit ugly I guess. Maybe it's fine as-is.